R/create_individual_file.R

```diff
@@ -795,3 +796,33 @@ join_slf_lookup_vars <- function(individual_file,
 
   return(individual_file)
 }
+
+#' Join sc client variables onto individual file
+#'
+#' @description Match on sc client variables.
+#'
+#' @param year financial year.
+#' @param individual_file the processed individual file
+#' @param sc_client SC client lookup
+#' @param sc_demographics SC Demographic lookup
+join_sc_client <- function(year,
+                           individual_file,
+                           sc_client = read_file(get_source_extract_path(year, "Client")),
+                           sc_demographics = read_file(get_sc_demog_lookup_path(),
+                             col_select = c("sending_location", "social_care_id", "chi")
+                           )) {
+  # Match to demographics lookup to get chi
+  join_client_demog <- sc_client %>%
+    dplyr::left_join(
+      sc_demographics,
+      by = c("sending_location", "social_care_id")
+    )
+
+  # Match on client variables by chi
+  individual_file <- individual_file %>%
+    dplyr::left_join(join_client_demog,
+      by = "chi"
+    )
```

Comment:
We don't want to include these in the indiv file - could you also check that there's no other variables we're not expecting please?
```suggestion
    ) %>%
     dplyr::select(!c("sending_location", "social_care_id"))
```